src/models/application-registration/index.js

```diff
@@ -7,4 +7,5 @@ export default Object.freeze({
   publicKey: PropTypes.string,
   repository: PropTypes.string.isRequired,
   sharedSecret: PropTypes.string.isRequired,
+  serviceAccountToken: PropTypes.string,
```

Comment:
I'm not sure how the whole feature is implemented, but this seems like a Real Bad Idea to have a "god mode" token passed around in every request for the application data — for every user. Again, I might not see the whole feature, but usually this sort of thing should be a "generate and read once" sort of feature; like GitHub or Azure do.